src/sage/algebras/fusion_rings/f_matrix.py

```diff
@@ -2427,7 +2427,7 @@ def certify_pentagons(self, use_mp=True, verbose=False):
             Partitioned 6 equations into 6 components of size:
             [1, 1, 1, 1, 1, 1]
             Computing appropriate NumberField...
-            sage: f.certify_pentagons()  is None      # not tested (long time ~1.5s, cypari issue in doctesting framework)
+            sage: f.certify_pentagons()  is None      # not tested, long time (~1.5s), cypari issue in doctesting framework
```

Comment:
```suggestion
            sage: f.certify_pentagons()  is None      # not tested (cypari issue in doctesting framework), long time (~1.5s)
```
If you want to change this one, then like this. The "cypari issue in doctesting framework" is explanation for the `not tested` tag, so it needs to go in parens.
(Actually also the "long time" there was just explanation, not a separate tag.)